java/arcs/android/crdt/ParcelableRawEntity.kt

```diff
@@ -67,10 +68,13 @@ data class ParcelableRawEntity(
                 collections[key] = set
             }
 
+            @Suppress("GoodTime") // use Instant
```

Comment:
why not instant? is the comment a todo (in that case can it be rephrased as a todo?)?

Reply:
iirc it's not cross platform, so i couldn't add it. this is an internal check, so we're just suppressing the error for now.

Reply:
can you say that as a comment in the code ("we are not using instant because ....")